openam-ui/openam-ui-ria/src/main/resources/templates/common/components/tab/TabComponentTemplate.html

```diff
@@ -2,7 +2,7 @@
     <ul class="nav nav-tabs" role="tablist">
         {{#each tabs}}
             <li>
-                <a href="#" data-tab-id="{{this.id}}" role="tab" data-toggle="tab" aria-expanded="false">{{this.title}}</a>
+                <a href="#{{this.id}}" data-tab-id="{{this.id}}" role="tab" data-toggle="tab" aria-expanded="false">{{this.title}}</a>
```

Comment:
Is this change necessary?

Reply:
It is, see https://jquery.com/upgrade-guide/3.0/#breaking-change-jquery-quot-quot-and-find-quot-quot-are-invalid-syntax. It would throw an error on tab click.